op3_motion_replay/src/op3_motion_replay.cpp

```diff
@@ -25,6 +31,24 @@ namespace robotis_op
 		
 	}
 	
+	void MotionReplay::enableModule(std::string module_name)
+	{
+		std_msgs::String msg;
+		msg.data = module_name;
+		
+		module_pub_.publish(msg);
+		
+		/*
+		robotis_controller_msgs::SetModule msg;
+		msg.request.module_name = module_name;
+		
+		if(!joint_module_.call(msg))
+		{
+			ROS_ERROR("module not set...\n");
+		}
+		*/

```

Comment:
should we remove this before merging to master? It appears to be dead test code, but it also appears to be something you are attempting to use still.  I will wait for an answer before approving or not.